sweetest/sweetest-base/src/main/java/com/mysugr/sweetest/internal/CommonBase.kt

```diff
@@ -0,0 +1,3 @@
+package com.mysugr.sweetest.internal
+
+interface CommonBase
```

Comment:
The name is very very generic. I have no idea what it's used for or why it's needed. It it possible to give it a more expressive name? Or ditch it if not needed?

Reply:
the final naming decisions are done in the upcoming PR, but i considered naming it `InternalTestElement`. that would be the type hierarchy i imagined:

```
InternalTestElement
  ApiTestElement (in v1)
    BaseJUnitTest
    BaseSteps
  ApiTestElement (in v2)
    BaseTest
      JUnit4Test (in junit)
    Steps
```